monai/apps/detection/networks/retinanet_network.py

```diff
@@ -293,19 +302,21 @@ def __init__(
         self.cls_key: str = "classification"
         self.box_reg_key: str = "box_regression"
 
-    def forward(self, images: Tensor) -> dict[str, list[Tensor]]:
+    def forward(self, images: Tensor) -> dict[str, list[Tensor]] | list[Tensor]:
```

Comment:
Hi @Can-Zhao ,
    The TorchScript doesn't support the type hint with `|` in the `forward` method (other methods are fine).  It will report  an error below. Could you please replace it with a `Union` or `Any`. `Union` would be better if it can pass the ci test.

```
Expression of type | cannot be used in a type expression:
  File "/home/liubin/data/github_monai/det-can/MONAI/monai/apps/detection/networks/retinanet_network.py", line 305
    def forward(self, images: Tensor) -> dict[str, list[Tensor]] | list[Tensor]:
                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
        """
        It takes an image tensor as inputs, and outputs predicted classification maps
```

Thanks,
Bin

Reply:
Sorry, just noticed that the `Union` type led  to a flake8 failure. Maybe `Any` is better.
Thanks,
Bin

Reply:
I changed it previously, but somehow reverted by a git pull. I just corrected it.

Reply:
Hi Can, were you converted it back to the `Union` type? The `Union` type will be converted to the `|` expression automatically by the `pre-commit.ci`. Could you please try the `Any` type as the return type?

Reply:
Ah! Did not know it. Just corrected it